### PR TITLE
UHM-7381: azcopy hanging HUM server?

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

**Review by cioan — APPROVED** [review]
Thanks @mogoodrich !

File: mirebalais-modules/openmrs/manifests/backup.pp [hunk 1]
```diff
@@ -66,7 +68,7 @@
 
   cron { 'backup-az':
     ensure  => present,
-    command => 'sh /usr/local/sbin/backupAzure.sh',
+    command => '/usr/bin/flock -n /home/tomcat/backups/cron.backupAzure.lock /usr/local/sbin/backupAzure.sh',
```
[the remark this answers is not in the document]
Comment (mseaton) [reply mirebalais-modules/openmrs/manifests/backup.pp:71]: Awesome

File: mirebalais-modules/openmrs/templates/backupAzure.sh.erb [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 export PATH=/usr/local/sbin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin
+export AZCOPY_CONCURRENCY_VALUE=<%= @azcopy_concurrency_value %>
+export AZCOPY_CONCURRENT_FILES=<%= @azcopy_concurrent_files %>
```
Comment (mseaton) [inline mirebalais-modules/openmrs/templates/backupAzure.sh.erb:4]: Setting these as environment variables is enough?  These will get picked up?

**Review by mseaton — APPROVED** [review]
Awesome work on this @mogoodrich

File: mirebalais-modules/openmrs/templates/backupAzure.sh.erb (hunk 2 above)
Comment (mogoodrich) [reply mirebalais-modules/openmrs/templates/backupAzure.sh.erb:4]: yes, that was how it was documented (and it worked)...

**Pull request merged**